pages/events.tsx

```diff
@@ -1,159 +0,0 @@
-import { useEffect, useRef, useState } from 'react';
-import { Layout } from 'components/layout';
-import { GetStaticProps } from 'next';
-import { useTina } from 'tinacms/dist/react';
-import { client } from '../tina/__generated__/client';
-import { Card } from 'components/blocks/Events/Events';
-import { DefaultSeo } from 'next-seo';
-
-const EventsPageHeader = ({ title, byline }) => {
-  return (
-    <div className="text-center mx-auto py-2">
-      <h1 className="pt-20 pb-5 text-4xl font-tuner bg-gradient-to-br from-orange-400 via-orange-500 to-orange-600 bg-clip-text text-transparent">
-        {title}
-      </h1>
-      <div>
-        <p className="text-gray-800 w-2/3 mx-auto break-words">{byline}</p>
-      </div>
-    </div>
-  );
-};
-
-const EventsPage = (props: { query: string; data: any; vars: any }) => {
-  const tinaData = useTina({
-    query: props.query,
-    data: props.data,
-    variables: props.vars,
-  });
-
-  const seoData = tinaData.data.eventsConnection.edges[0].node.seo
-
-  const events = tinaData.data.eventsConnection?.edges.map(
-    (edge: any) => edge.node
-  );
-  const eventsData = events[0];
-
-  const now = new Date();
-
-  //Filtering and sorting by time to go (ASC)
-  const upComingEvents = eventsData.cardItems
-    .filter((event) => {
-      const eventStartDate = new Date(event.startDate);
-      return eventStartDate.getTime() >= now.getTime();
-    })
-    .sort(
-      (a, b) =>
-        new Date(a.startDate).getTime() - new Date(b.startDate).getTime()
-    );
-
-//Filter and sorting by time passed (ASC)
-  const pastEvents = eventsData.cardItems
-    .filter((event) => {
-      const eventStartDate = new Date(event.startDate);
-      return eventStartDate.getTime() < now.getTime();
-    })
-    .sort(
-      (a, b) =>
-        new Date(b.startDate).getTime() - new Date(a.startDate).getTime()
-    );
-
-  const [visibleCards, setVisibleCards] = useState<string[]>([]);
-
-  const handleIntersection = (entries: IntersectionObserverEntry[]) => {
-    entries.forEach((entry) => {
-      if (entry.isIntersecting) {
-        const id = entry.target.getAttribute('data-id');
-        if (id && !visibleCards.includes(id)) {
-          setVisibleCards((prev) => [...prev, id]);
-        }
-      }
-    });
-  };
-
-  useEffect(() => {
-    const observer = new IntersectionObserver(handleIntersection, {
-      threshold: 0.2, // Trigger animation when 20% of the card is visible
-    });
-
-    const cards = document.querySelectorAll('.event-card');
-    cards.forEach((card) => observer.observe(card));
-
-    return () => observer.disconnect();
-  }, [visibleCards]);
-
-  return (
-    
-    <Layout>
-      <DefaultSeo
```

Comment:
I couldn't find where did you configured seo in App Router.
This can be handled using `generateMetadata` function - https://nextjs.org/docs/app/building-your-application/optimizing/metadata.

Add `generateMetadata` to `page.tsx` file in App Router to set seo fields